build-system/conformance-config.textproto

```diff
@@ -248,3 +248,11 @@ requirement: {
   whitelist: 'src/service/cid-api.js'
   whitelist: 'src/service/cid-impl.js'
 }
+
+# Function
+
+requirement: {
+  type: BANNED_PROPERTY_CALL
+  error_message: 'Function.prototype.name is not allowed due to IE11 non-support.'
```

Comment:
Do we usually explain the reasons why we ban specific properties? By shaming specific browsers? :stuck_out_tongue_closed_eyes:

Reply:
I figure some context helps so we can un-ban if/when we drop IE11 support.

Reply:
Makes sense.